extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -72,8 +88,16 @@ export function RemoteQueries(): JSX.Element {
         ({queryResult.executionDuration}), {queryResult.executionTimestamp}
       </p>
       <p className="vscode-codeql__paragraph">
-        <span className="vscode-codeql__query-file">{octicons.file} <span>{queryResult.queryFile}</span></span>
-        <span>{octicons.codeSquare} <span>query</span></span>
+        <span className="vscode-codeql__query-file">{octicons.file}
+          <a className="vscode-codeql__query-file-link" href="#" onClick={openQueryFile}>
```

Comment:
Sorry I didn't think about this before pressing "Approve", but I think these may need some styling tidying now that they become links (so that it's like normal text and the visited state doesn't affect the colour).

Or, perhaps more semantically correct if they're buttons rather than links?

Reply:
Oops, good point, thanks for spotting that! I've added some styling, which keeps the text the same colour in all situations.